.release-please-manifest.json

```
@@ -103,5 +103,6 @@
   "plugins/source/bitbucket+FILLER": "0.0.0",
   "plugins/source/notion": "1.1.8",
   "plugins/source/xkcd": "1.4.4",
-  "plugins/transformer/basic": "1.1.5"
+  "plugins/transformer/basic": "1.1.5",
+  "plugins/transformer/json": "1.0.0"
```

```suggestion
  "plugins/transformer/basic": "1.1.5"
```

This is not needed, our bot updates it when it releases a new version. By putting a version here you're telling the bot the existing released version is `1.0.0` so the next one will be `1.1.0`